docs/sdk/server-side-sdks/php/php-install.md

```diff
@@ -38,4 +38,54 @@ Download the files and include `autoload.php`:
 require_once('/path/to/DevCycle/vendor/autoload.php');
 ```
 
+## Local Bucketing Proxy
+
+Due to complexities with the PHP application lifecycle and state management, local bucketing functionality requires a different approach compared to the other server SDKS.
+
+To access this functionality in PHP, DevCycle provides a **proxy** process that can run alongside your PHP application or on a separate host in your environment. This proxy mimics the Cloud Bucketing API but provides for higher speed variable evaluations due to the reduction of network latency and config caching; all powered by DevCycle's high performance Go Server SDK.
+
+The local bucketing proxy can be downloaded as a binary or package from here: https://github.com/DevCycleHQ/local-bucketing-proxy/releases
+
+#### Proxy Configuration
+
+Once the proxy is downloaded and installed, it can be configured via either environment variables or command line arguments. See the [README](https://github.com/DevCycleHQ/local-bucketing-proxy#options) for the configuration options.
+
+At a minimum, you will need set the `DVC_LB_PROXY_SDK_KEY` environment variable to activate the proxy. This should be set to your SDK key for your PHP application.
+The rest of the values can be left unchanged for basic operation. This will default to starting an HTTP server on port 8080.
```

Comment:
@chris-hoefgen / @rob-odwyer should we use a different default port? isn't `8080` commonly used by a lot of systems?

Reply:
yeah, we could switch that out. while it likely won't conflict with an php servers when we branch out to .net and java it will definitely conflict with default ports for many frameworks

Reply:
talked with rob and I don't think it is critical to update the default port but I explicitly added a following note of which env var to set to change it.